src/content/en/fundamentals/instant-and-offline/get-started/textcontent-3.md

```diff
@@ -0,0 +1,276 @@
+already been individually compressed generally provides no significant size reduction.
+
+Unlike desktop or browser-based local compression, Gzip works at the server to identify and 
+process certain file types that you specify. While all modern browsers support Gzip compression 
+for HTTP requests, you must properly configure your server to deliver the compressed resource 
+when it is requested. Different server types, of course, have different setup requirements. 
+You configure the Apache server, for example, via the `.htaccess` file, which would include 
+something like this.
+
+```
+<IfModule deflate_module>
+    # Enable compression for the following file types
+    AddOutputFilterByType        	\
+     DEFLATE                     	\
+      application/javascript     	\
+      text/css                   	\
+      text/html                  	\
+      text/javascript            	\
+      text/plain                 	\
+      text/xml
+</IfModule>
+```
+
+BetterExplained has a very good 
+[article on Gzip compression](https://betterexplained.com/articles/how-to-optimize-your-site-with-gzip-compression/), 
+including background information, examples, and caveats.
+
+So, having enabled GZip on your server, how do you know whether it's actually serving 
+compressed files? An easy way to find out is to check it at GIDNetwork's Gzip test site.
+
+[http://www.gidnetwork.com/tools/gzip-test.php](http://www.gidnetwork.com/tools/gzip-test.php)
+
+The report includes general information about site compression, an interesting "What if" 
+chart that shows how much compression the site would gain at various Gzip compression levels, 
+and the response headers and page source. 
+
+Here, we ran the test on GIDNetwork's own root URL -- interestingly, it did not appear 
+to be compressed.
+
+![GIDNetwork](images/image_303.png)
+
+Gzip can further compress already-compressed files, and that's a valid approach. In fact, 
+to get the greatest compression ratio for text-based resources, first minify them individually 
+prior to deployment and then compress them at delivery via a Gzip-enabled server.
+
+## Reduce Library Use
+
+Popular CSS and JavaScript libraries do their best to minify and compress their download files, 
+but generally they're still pretty serious bandwidth consumers. jQuery, for example -- depending 
+on the version and the compression algorithms applied -- might range from 28k to over 250k. 
+If you need most of a given library's features, fine; but if you only need one or two specific 
+things, you can save a lot of download time by replacing those features with single-use 
+functions or CSS rules.
+
+For example, a website might use jQuery's handy `slideToggle` feature to expand and collapse 
+a div, like this.
+
+`<a href="javascript:void(0)" onclick="$('#more').slideToggle()">More...</a>`
+
+Yes, it works great and it's easy to code, but jQuery is a lot of download overhead for one text 
```

Comment:
Rather than using this reference from W3Schools (and the CSS transition example), I would personally just use a reference from http://youmightnotneedjquery.com/#toggle_class (pretty similar use-case) and just use that instead. Reading through this a few times, I feel that would get the same point across while feeling a little less verbose.